website/content/docs/enterprise/license/index.mdx

```diff
@@ -21,8 +21,8 @@ for more details.
 
 ## Expiring Licenses
 
-Nomad Enterprise license have an expiration time. You can read the license on
-a server with the `nomad license get` command:
+Nomad Enterprise licenses have an expiration time and a termination time. You
+can read the license on a server with the `nomad license get` command:
```

Comment:
fun tidbit: `license get` does not show TerminationTime

https://github.com/hashicorp/nomad/blob/v1.5.3/command/license_get.go#L61-L67
=> https://github.com/hashicorp/nomad/blob/v1.5.3/command/license.go#L54

I imagine it just got overlooked in the past.  Do we want to add that too?

Reply:
Probably. I can do that under separate PR; this should just be a docs PR.

Reply:
Done in https://github.com/hashicorp/nomad/pull/16892